bindings/matrix-sdk-ffi/src/session_verification.rs

```diff
@@ -60,80 +73,128 @@ pub struct SessionVerificationController {
 
 #[matrix_sdk_ffi_macros::export]
 impl SessionVerificationController {
-    pub async fn is_verified(&self) -> Result<bool, ClientError> {
-        let device =
-            self.encryption.get_own_device().await?.context("Our own device is missing")?;
+    pub fn set_delegate(&self, delegate: Option<Box<dyn SessionVerificationControllerDelegate>>) {
```

Comment:
I considered switching away from a delegate and towards a listener but the public api changes are so small that I don't really thing it's worth it at this point. Let's reassess when we start thinking about user verification.